docs/docs/components/quickstart/nodejs/sources/README.md

```diff
@@ -495,17 +495,19 @@ let axios = require("axios");
 Next, let's add an **app prop**, which will enable us to use Pipedream managed auth with this component. For this example, we'll add Github:
 
 ```javascript
-github: {
-  type: "app",
-  app: "github",
-}
+props: {
+  github: {
+    type: "app",
+    app: "github",
+  },
+},
 ```
 
 **IMPORTANT: The CLI will prompt you to select a connected account (or connect a new one) when you deploy (or update) this component.**
 
 > **Note:** The value for the `app` property is the name slug for the app in Pipedream. This is not currently discoverable, but it will be in the near future. For the time being, if you want to know how to reference an app, please reach out on our public Slack.
 
-Finally, we'll update the `run()` method to fetch issues from Github using `axios` and emit them. Notice that we're passing the `oauth_access_token` in the authorization header by referencing the app prop `this.github.$auth.oauth_access_token`. Also, similar to the RSS example, it's important that you use the `pddemo/demo` repo so you can test the next dedupe strategy.
+Finally, we'll update the `run()` method to fetch issues from Github using `axios` and emit them. Notice that we're passing the `oauth_access_token` in the authorization header by referencing the app prop `this.github.$auth.oauth_access_token`. Also, similar to the RSS example, it's important that you use `https://api.github.com/repos/pddemo/demo/issues` as the repo so you can test the next dedupe strategy.
```

Comment:
I see why you made this change, but technically `pddemo/demo` is still the name of the repo, so I think we should keep that part as-is. Maybe modify to something like:

> Also, similar to the RSS example, it's important that you use the `pddemo/demo` repo so you can test the next dedupe strategy (see the `url` below).

to clarify that the reader should pay attention to the URL?

Reply:
I think I was thrown off by the specific reference to the RSS example. I was looking at pddemo/demo and thinking, "Have I seen this before? Was this in the last example?" Reading further to the sample code cleared it up but there was definitely a moment of confusion there!

How about this wording?

> Again, it's important that you stick with the pddemo/demo repo shown in the below example so you can test the next dedupe strategy.

Reply:
I like it!

Reply:
Great! Everything should be good to go then.